src/Conversion/KrnlToLLVM/KrnlEntryPoint.cpp

```diff
@@ -236,16 +236,8 @@ class KrnlEntryPointOpLowering : public OpRewritePattern<KrnlEntryPointOp> {
 
     Value numOutput =
         create.llvm.constant(int64Ty, (int64_t)outMemRefList.size());
-
-    auto mallocSym = getOrInsertMalloc(rewriter, module);
-    // TODO(tjingrant): get pointer size from data layout.
-    size_t kPtrSize = 8;
-    Value outputOmtPtrsArraySizeInByte = create.llvm.constant(
-        int64Ty, (int64_t)(outMemRefList.size() * kPtrSize));
-    Value outOmtPtrsArr = create.llvm.call(
-        LLVM::LLVMPointerType::get(IntegerType::get(module.getContext(), 8)),
-        mallocSym, ArrayRef<Value>(outputOmtPtrsArraySizeInByte));
-    outOmtPtrsArr = create.llvm.bitcastI8PtrPtr(outOmtPtrsArr);
+    Value outOmtPtrsArr = create.llvm._alloca(
```

Comment:
I assume that you lifted this pattern from elsewhere. Pointers are typically aligned at a multiple of 8 byes ... I assume that there is something like "aligned at natural data size" in that logic. Meaning that you have seen other pointer arrays being allocated like that.

Reply:
@cjvolzka As you can see the old code assumes the pointer size is 8, so it would be helpful to add a comment saying "Assume that OMTensor pointer size is 8", because the allocation size is potentially different from the actual size for the tensor list. The comment helps us debug the code in the future.

Reply:
@AlexandreEichenberger correct. I lifted the pattern from a few lines above it actually (line 204):
```
      Value ptrToMemRef =
          create.llvm._alloca(memRefPtrTy, one, /*alignment=*/0);
```